scss/partials/_cards.scss

```diff
@@ -102,5 +111,10 @@
     .content-block__title {
       @extend .a;
     }
+    img {
```

Comment:
As a BEM standard, we should never call out root element selectors (outside of our base partial), always classes. The only exception might be with the blog, if we're needing to do some convention breaking to support the user entered WYSIWYG content that won't have classes coded in.